src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1982,6 +1978,16 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
         return results;
     }
 
+    private void showAllQuestions(UserRole role, Map<String, String> params,
```

Comment:
We will prefer it private methods knew the minimum of what they need to know in general, instead of passing in the full UserRole, in this context, we can simply pass in a boolean isUserRoleInstructor and for params, doesParamsContainQuestionId for example